topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -0,0 +1,1182 @@
+>    >
+>    > 	recipes/bellerophon/
+>    >
+>    > no changes added to commit (use "git add" and/or "git commit -a")
+>    > ```
+>    {: .code-out}
+>
+> 2. Add your recipes folder, commit, and push it.
+>
+>    > ### {% icon code-in %} Input: Bash
+>    > ```bash
+>    > git add recipes/bellerophon
+>    > git commit -m 'Add recipe for bellerophon 1.0'
+>    > git push origin -u bellerophon_bioconda
+>    > ```
+>    {: .code-in}
+>
+>    Git will give you a url for creating the pull request, which can be clicked to be taken to github.
+>
+>    ![Git push](../../images/git_push.png "Expected output of the `git push` command")
+>
+> Once on the github pull request page, we can write a short description of the recipe we want to merge, and click the button.
+>
+> ![Github PR](../../images/create_pr.png "An open pull request for the bellerophon conda recipe")
+{: .hands_on}
+
+
+
+## Deploying a Conda Package
+
+Once our new recipe has been merged, installing the package should be fairly painless. Galaxy, if configured to do so,
+will automatically install any conda dependencies a tool asks for, whereas manual installation can be as easy as using
+the `conda create` command to install python, bellerophon, and pysam into a conda environment named `bellerophon`. The
+`-y` flag tells conda not to ask for confirmation before installing the software and its dependencies into the named
+environment.
+
+```bash
+conda create -y --name bellerophon bellerophon=1.0 -c conda-forge -c bioconda
+```
+
+# Galaxy Tool Wrappers
+
+Galaxy tool wrappers convert the available command line inputs and parameters of a given tool into a grapical
+user interface, then translate the user's selections back into the command to be run. This section will focus
+on writing a wrapper. Although this section will cover the full process and many of the options available to
+anyone wrapping a tool, a more complete list is available in [the Galaxy tool docs](https://docs.galaxyproject.org/en/latest/dev/schema.html).
+
+## Initializing a Tool Wrapper
+
+Initializing a tool wrapper to be run in Galaxy is simple
+
+1. Go to the "tools" directory in your Galaxy instance.
+2. Make a new directory for tools you will be wrapping.
+3. Use `planemo` to initialize a new tool wrapper with basic formatting.
+    - Here we will be using bellerophon, but the parameters can be set for any tool.
+    - The tool ID is the internal reference id for the tool, and may contain no spaces or capitalized letters
+    - The tool name is the text that will be visible to the user.
+
+    ```bash
+    cd galaxy/tools
+    mkdir -p myTools/bellerophon
+    cd myTools/bellerophon
+    planemo tool_init --id bellerophon --name bellerophon
+    ```
+
+4. You can now open the new xml file that has been generated to begin the wrapping. It should look something like this:
+![Bellerophon initial wrapper](../../images/tool_init.png "Initial tool wrapper after running planemo tool_init")
```

Comment:
could this be replaced with the raw text rather than a screenshot?

Reply:
I had wanted to show it in an editor, but we can do raw text if you'd prefer

Reply:
a code block with text would be more accessible for non-sighted users. Is there important content inside the editor? Looking at the image it looked like mostly just the content of the xml, syntax highlighted but this would only compare to a user's view if they also used the same colour scheme, and syntax highlighting for XML is built into the markdown renderer we use? If that helps

Reply:
Makes sense. I'll switch it out now